Probesys/bileto
# new: Allow to search contracts

## Related issue(s)



https://github.com/Probesys/bileto/issues/819

## How to test manually



- Create a bunch of contracts
- Try the different options to search contracts from the syntax help
- Verify that the results are coherent

## Checklist
